### ISSUE TITLE
Random Joke Generator from API

### ISSUE THREAD

**Issue by Ayonijakaushik19** [LetsGrowMoreCommunity/WebDev-Playyard#26]
Hi, @SnehaAgarwal361 I want to add this project to this repo. Please assign this to me.

**Comment by SnehaAgarwal361** [comment]
Yes go for it please take care of the contribution guidelines and do not forget to add a short video and ss of your work.
Happy coding:)

**Comment by SnehaAgarwal361** [comment]
Hey @Ayonijakaushik19 is your project ready?
